api/lambdas/withings/index.js

```diff
@@ -54,7 +54,10 @@ const lookup = async address => {
 async function forwardCallToServer(req) {
   console.log(`Verified! Calling server...`);
 
-  const resp = await api.post(apiServerURL, req.body, { headers: req.headers });
+  const convertParams = new URLSearchParams(req.body);
+  console.log(`convertParams - body: ${convertParams}`);
```

Comment:
NABD, but we could have a single log with the both server URL and the body... just move the `Calling server` right before calling it and add the data there